src/main/java/io/jenkins/plugins/services/impl/HttpClientWikiService.java

```diff
@@ -104,10 +104,8 @@ public String cleanWikiContent(String content) throws ServiceException {
       return null;
     }
     final Element wikiContent = elements.first();
-    // Remove H4 tag with Plugin information, that is part of the jenkins-plugin-info macro
-    wikiContent.getElementsContainingOwnText("Plugin Information").tagName("h4").remove();
-    // Remove any Confluence tables
-    wikiContent.getElementsByClass("confluenceTable").remove();
+    // Remove any Confluence stuff at the top
+    wikiContent.getElementsByClass("conf-macro output-inline").first().remove();
```

Comment:
The macro can be anywhere on the page; some plugins have a table of contents or similar first (e.g. https://wiki.jenkins.io/display/JENKINS/Accelerated+Build+Now+Plugin) so be careful with something like this.

If possible, look for a table with known contents (e.g. first cell has "Plugin ID" text etc.) and remove that.